navigation/MainTabNavigator.js

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-import { StyleSheet, StatusBar, View } from 'react-native';
+import { StyleSheet, StatusBar, View, Platform } from 'react-native';
 import { createStackNavigator } from 'react-navigation-stack';
-import { createMaterialTopTabNavigator } from 'react-navigation-tabs';
+import { createBottomTabNavigator } from 'react-navigation-tabs';
```

Comment:
i forgot why i used the `MaterialTopTabNavigator`, but `BottomTabNavigator` works just fine 🤷‍♂